user_guide_src/source/libraries/cookies/017.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use CodeIgniter\Cookie\Cookie;
+use Config\Services;
+use DateTime;
```

Comment:
CI4 has a class `use CodeIgniter\I18n\Time;`. These documents are codeigniter, I think it is better to use the internal library.
Just a personal preference.

Reply:
Changed to use undocumented `max-age`.